web/packages/design/src/SyncStamp/SyncStamp.test.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { render, screen } from 'design/utils/testing';
```

Comment:
```suggestion

import { render, screen } from 'design/utils/testing';
```

Not sure if it matters in TypeScript, but in Go we always want to leave an empty line between the license and the line of code so that IDEs don't think that the license is a comment for the code.